tests/lib/rules/jsx-uses-react.js

```diff
@@ -35,50 +35,50 @@ const settings = {
 // -----------------------------------------------------------------------------
 
 const ruleTester = new RuleTester({ parserOptions });
-const linter = ruleTester.linter || eslint.linter || eslint.Linter;
-linter.defineRule('jsx-uses-react', require('../../../lib/rules/jsx-uses-react'));
+const ruleDefiner = ruleTester[Symbol.for('react.RuleTester.RuleDefiner')] || ruleTester.linter || eslint.linter || eslint.Linter;
```

Comment:
```suggestion
const ruleDefiner = (typeof Symbol !== 'undefined' && Symbol.for && ruleTester[Symbol.for('react.RuleTester.RuleDefiner')]) || ruleTester.linter || eslint.linter || eslint.Linter;
```
is probably wise, just in case

(also it'd probably be good to make a helper for `ruleDefiner`, so it can be used in both places)

Reply:
Done in https://github.com/jsx-eslint/eslint-plugin-react/pull/3759/commits/b0f5afd085e033c99a70d8bcd1dcb7e04fe7cda5. I added `tests/helpers/getRuleDefiner.js`, similar to the existing `tests/helpers/getESLintCoreRule.js`.